pycheribuild/projects/cross/muslc.py

```diff
@@ -101,5 +101,5 @@ def setup(self) -> None:
         # FIXME: Morello muslc does not address warnings with implicit functions defined
         # and fails building. This should be fixed in Morello Busybox codebase manually
         # or when they update to recent revisions/releases
-        self.cross_warning_flags.append("-Wno-error=implicit-function-declaration")
+        self.cross_warning_flags.clear()
```

Comment:
Note: there's now a merged fix for the bad function pointer cast: https://git.morello-project.org/morello/musl-libc/-/merge_requests/154